docs/learn/dapp-staking-v3/dapp-staking-faq.md

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_position: 3
```

Comment:
Seems strange FAQ comes before actual chapter about dApp staking v3.

Reply:
already changed this but updated to be not 2 and 3 but 1 and 2 positions